src/components/MultiHopTrade/components/LimitOrder/components/LimitOrderInput.tsx

```diff
@@ -376,7 +383,6 @@ export const LimitOrderInput = ({
   const isLoading = useMemo(() => {
     return (
       isCheckingAllowance ||
-      isLimitOrderQuoteFetching ||
```

Comment:
Not relevant to general-purpose loading state anymore, only for the button loading/disabled state now